mathjax3-ts/input/tex/cancel/CancelConfiguration.ts

```diff
@@ -0,0 +1,87 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2018 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/**
+ * @fileoverview Configuration file for the cancel package.
+ *
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import {Configuration} from '../Configuration.js';
+import TexParser from '../TexParser.js';
+import {TexConstant} from '../TexConstants.js';
+import {CommandMap} from '../SymbolMap.js';
+import {ParseMethod} from '../Types.js';
+import ParseUtil from '../ParseUtil.js';
+import {ENCLOSE_OPTIONS} from '../enclose/EncloseConfiguration.js';
+
+
+// Namespace
+export let CancelMethods: Record<string, ParseMethod> = {};
+
+
+/**
+ * Parse function for cancel macros of the form \(b|x)?cancel[attributes]{math}
+ * @param {TexParser} parser The current tex parser.
+ * @param {string} name The name of the calling macro.
+ * @param {string} notation The type of cancel notation to use.
+ */
+CancelMethods.Cancel = function(parser: TexParser, name: string, notation: string) {
+  const attr = parser.GetBrackets(name, '');
+  const math = parser.ParseArg(name);
+  const def = ParseUtil.keyvalOptions(attr, ENCLOSE_OPTIONS);
+  def['notation'] = notation;
+  parser.Push(parser.create('node', 'menclose', [math], def));
+};
+
+
+/**
+ * Parse function implementing \cancelto{value}[attributes]{math}
+ * @param {TexParser} parser The current tex parser.
+ * @param {string} name The name of the calling macro.
+ * @param {string} notation The type of cancel notation to use.
+ */
+
+CancelMethods.CancelTo = function(parser: TexParser, name: string) {
+  const attr = parser.GetBrackets(name, '');
+  let value = parser.ParseArg(name);
+  const math = parser.ParseArg(name);
+  const def = ParseUtil.keyvalOptions(attr, ENCLOSE_OPTIONS);
+  def ['notation'] = TexConstant.Notation.UPDIAGONALSTRIKE + ' ' +
+    TexConstant.Notation.UPDIAGONALARROW;
```

Comment:
In the old days, MathML didn't have a notation for the arrow used by `\cancel`, and so MathJax used the (non-standard) `updiagonalarrow` here (and the output jax implemented it).  but since other renderers didn't know about it, we added `updiagonalstrike` as well so that if used in native MathML, there would be at least some indication of the arrow.

Since then, MathML has added a number of arrows, and I believe the correct one is `northeastarrow`, so we should change that here (it may need to be added to `TexConstant.Notation`).  It has been long enough that I would be happy to remove the `updiagonalstrike` and just use `northeastarrow`, but I'm OK if you want to leave it for backward compatibility.

Reply:
For the record, official MathML notation is `longdiv, actuarial, phasorangle, radical, box, roundedbox, circle, left, right, top, bottom, updiagonalstrike, downdiagonalstrike, verticalstrike, horizontalstrike, northeastarrow, madruwb, text`.

Reply:
OK, total confusion.... 
https://www.w3.org/TR/MathML2/chapter3.html#presm.menclose
gives has `northeastarrow` but no `updiagonalarrow`.

This site
https://www.tutorialspoint.com/mathml/mathml_enclosing.htm
(first hit on DuckDuckGo when including `northeastarrow` in search) has `northeastarrow` but no `updiagonalarrow` and a peculiar `text` element.

Finally the Moziall docs (first hit on DDG with only menclose:
https://developer.mozilla.org/en-US/docs/Web/MathML/Element/menclose
has `updiagonalarrow` but not `northeastarrow`.

Once again MathML is not a standard but a mess. 

We should retain `updiagonalarrow`.